app/build.gradle

```diff
@@ -48,6 +48,8 @@ dependencies {
     implementation Dependencies.design
     implementation Dependencies.constraint_layout
     implementation Dependencies.appCompat
+    implementation 'androidx.appcompat:appcompat:1.0.2'
```

Comment:
I think these both dependencies, are already here in form of variables, so these may be repeated